pull_request_template.md

```diff
@@ -1,8 +1,8 @@
-<!-- Please fill in the **bold** fields and tick all applicable boxes in the "Preview" tab. -->
+<!-- Please fill in the **bold** fields and tick all applicable boxes by placing an "x" inside "[ ]". -->
```

Comment:
The preview tab doesn't allow you to tick the boxes, that's why I removed it. I think it's better to have harder instructions than wrong one.